packages/checkout/discounts-meta/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { createSlotFill } from '../slot';
+
+const slotName = '__experimentalDiscountsMeta';
+
+const {
+	Fill: ExperimentalDiscountsMeta,
+	Slot: DiscountsMetaSlot,
+} = createSlotFill( slotName );
+
+const Slot = ( { className, extensions, cart, components, contexts } ) => {
```

Comment:
Context was deleted as far as I remember.

Reply:
Yep, you're right it was. Removing 🗑️